src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -171,6 +171,21 @@ public void TestNormalizeStatement1()
             TestNormalizeStatement("Func<string, int> f = blah;", "Func<string, int> f = blah;");
         }
 
+        [Theory]
+        [InlineData("int*p;", "int* p;")]
+        [InlineData("int *p;", "int* p;")]
+        [InlineData("int*p1,p2;", "int* p1, p2;")]
+        [InlineData("int *p1, p2;", "int* p1, p2;")]
+        [InlineData("int**p;", "int** p;")]
+        [InlineData("int **p;", "int** p;")]
+        [InlineData("int**p1,p2;", "int** p1, p2;")]
+        [InlineData("int **p1, p2;", "int** p1, p2;")]
```

Comment:
Consider also testing some function pointers (`delegate*<int, void> p1;`) while you're here.

Reply:
@jcouv This PR doesn't handle function pointers. I'm not sure what the test result would be. But I'd expect this test case to fail.

I've added it to confirm from CI (sorry that I can't confirm locally).